client/utils/loadComponent.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+
+const loadComponent = (importCall) => {
```

Comment:
I'll update the style guide to reflect two line spaces after import statements. Kindly add an extra line space here